### PR TITLE
Add `alignment: soft;` to YAML

### PR DESCRIPTION
**Pull request by kpu**
This is required for HTML support to work.

### CODE REVIEW

**Comment by andrenatal** [comment]
Which version of bergamot-translator is required to have for this flag? The one we are currently using is compatible with this? I'll let @abhi-agg review this

**Comment by kpu** [comment]
It's been used in the demo since https://github.com/browsermt/bergamot-translator/blame/feb9c90429fe23423dc23c56e8d0ee19d85acec7/wasm/test_page/js/worker.js#L187

**Comment by kpu** [comment]
The HTML feature has always depended on this.  So any quality of HTML you see in the extension until this is merged would indeed be terrible.  I think we should start enforcing it with an error.

**Review by abhi-agg — APPROVED** [review]

**Comment by abhi-agg** [comment]
@andrenatal I verified that the artifacts that the extension currently uses already included this change. So, we are safe to merge it.

**Comment by andrenatal** [comment]
@abhi-agg does this fixes #51 for free?

**Pull request merged**
